ocm-container.sh

```diff
@@ -39,6 +40,8 @@ while [ "$1" != "" ]; do
                             ;;
     -x | --debug )          set -x
                             ;;
+    -dp | --disable-console-port )   DISABLE_CONSOLE_PORT_MAP=true
```

Comment:
Can we keep this flag as a single letter, so instead of `-dp` make it something like `-p` or something?  One day I'd love to see this get ported to go/cobra and I'd like the port to be backwards compatible.

Reply:
Sure, I was trying to use a flag that wouldn't conflict with docker/podman flags, but I can find something.